tracker/urls.py

```diff
@@ -22,6 +23,7 @@
 urlpatterns = [
     path('django-admin/', admin.site.urls),
 
+    path('api/v2/', include(api_urls)),
```

Comment:
Maybe we should put this at `api/beta/` or `api/edge/` for the moment so as not to make promises about its permanence?